pyproject.toml

```diff
@@ -23,14 +23,16 @@ whist-server = "whist_server.cli:main"
 
 [tool.poetry.dependencies]
```

Comment:
please use `^major.minor` syntax and use `[tool.poetry.group.dev.dependencies]` for all dev dependencies acccording to https://python-poetry.org/docs/managing-dependencies/#dependency-groups

Don't forget to do `poetry update` and `poetry install` as well ;)

Reply:
Done